rollup.config.js

```diff
@@ -49,13 +76,21 @@ export default [
   /* ESM bundle with externals */
   {
     input,
-    output: {
-      file: 'dist/bson.esm.js',
-      format: 'es'
-    },
+    output: [
+      {
+        file: 'dist/bson.esm.js',
+        format: 'esm',
+        exports: 'named',
+        sourcemap: true
+      }
+    ],
     // Notice the external buffer, and preferBuiltins false,
     // this bundle doesn't pull in 'buffer' or Map from 'core-js'
     external: ['buffer'],
-    plugins: [nodeResolve({ preferBuiltins: true }), commonjs()]
+    plugins: [
+      typescript({ tsconfig: './tsconfig.rollup.json' }),
+      nodeResolve({ preferBuiltins: true }),
```

Comment:
I just noticed the comment above saying "note preferBuiltins false" which not actually the case here

Reply:
Oops yea out of date comment, good catch.